projects/ecfuzzer/project.yaml

```diff
@@ -0,0 +1,2 @@
+homepage: "https://github.com/catenacyber/ecfuzzer"
```

Comment:
The project deserves a top-level readme. 
Also, maybe rename it to something containing the word `differential`?